python/tvm/topi/cuda/batch_matmul.py

```diff
@@ -161,7 +161,8 @@ def batch_matmul_cublas(cfg, x, y, out_shape=None):
     """
     b, m, k = x.shape
     b, n, k = y.shape
-    cfg.add_flop(b * m * k * n * 2)
+    if all([isinstance(s, int) for s in [b, m, n, k]]):
```

Comment:
The type of tensor shape is `Array<Integer>` so `s` could be `IntImm`.

Reply:
Done